src/views/evcharger/EVChargerControls.vue

```diff
@@ -517,7 +517,8 @@ export default defineComponent({
       evChargerSystemPointE,
       isSimpleModeRef,
       isRealDevices,
-      isRHE
+      isRHE,
+      isChargingSetShow: computed(() => store.getters.data(evChargerSystem.value.evCharger.ip, evChargerSystem.value.evCharger.eoj, 0x9E).indexOf(231) !== -1),
```

Comment:
@justdecodeme ,

Please name that variable better so other developers can easily understand its purpose.
e.g., `isChargeAmountVisible`